src/game_inventory.cpp

```diff
@@ -1426,10 +1426,11 @@ static item_location autodoc_internal( player &u, player &patient,
     } else if( patient.has_bionic( bionic_id( "bio_painkiller" ) ) ) {
         hint = _( "<color_yellow>Patient has Sensory Dulling CBM installed.  Anesthesia unneeded.</color>" );
     } else {
-        std::vector<const item *> a_filter = u.crafting_inventory().items_with( []( const item & it ) {
+        const inventory &crafting_inv = g->u.crafting_inventory();
```

Comment:
It looks intentional that this uses the passed-in player reference u instead of g->u, why reverse it?
```suggestion
        const inventory &crafting_inv = u.crafting_inventory();
```

Reply:
Thanks. That was a case of copy-paste in its terminal stage.